optimize/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -184,7 +184,7 @@ Define a secured connection to be able to communicate with a secured Elasticsear
 
 These settings are only relevant when operating Optimize with OpenSearch.
 
-#### Connection settings
+#### Connection settings OpenSearch
```

Comment:
I need to change this text so that I can link to this directly, otherwise the link will go to the heading with the same name under `ElasticSearch`

Reply:
I went ahead and standardized all these headings/updated the link, so this should stand out a little less now.

Reply:
Thanks, that is much cleaner